core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/scheduling/TestScheduleManager.java

```diff
@@ -241,6 +243,25 @@ public void testDescheduleRunningJob() throws InterruptedException
         assertThat(myScheduler.getQueueSize()).isEqualTo(0);
     }
 
+    @Test
+    public void testGetCurrentJobStatus() throws InterruptedException
+    {
+        CountDownLatch latch = new CountDownLatch(1);
+        UUID jobId = UUID.randomUUID();
+        ScheduledJob testJob = new TestScheduledJob(
+                new ScheduledJob.ConfigurationBuilder()
+                        .withPriority(ScheduledJob.Priority.LOW)
+                        .withRunInterval(1, TimeUnit.SECONDS)
+                        .build(),
+                jobId,
+                latch);
+        myScheduler.schedule(testJob);
+        new Thread(() -> myScheduler.run()).start();
+        Thread.sleep(100);
```

Comment:
Is this really necessary? To wait 100 seconds in just one unit test, try to reformulate this test to avoid this high number please